radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/UnverifiedVertex.java

```diff
@@ -155,8 +160,6 @@ public static UnverifiedVertex create(
 		List<Txn> txns,
 		BFTNode proposer
 	) {
-		Objects.requireNonNull(qc);
-
 		if (view.number() == 0) {
 			throw new IllegalArgumentException("Only genesis can have view 0.");
```

Comment:
Maybe should be moved to main constructor to keep validation in one place?

Reply:
done

Reply:
Rolled back. The proposed change breaks a number of tests.